server/assets/embed.go

```diff
@@ -67,3 +66,17 @@ func fromPath(path string) (assets fs.FS, err error) {
 
 	return
 }
+
+func FileNames(aPath string) (fileNames []string) {
```

Comment:
Minor thing but you could probably name `fileNames` to just `names`; just a thought, no need to change